html/index.html

```diff
@@ -17,8 +17,8 @@
                 <li><a href="purchases.html">Purchases</a></li>
             </ul> 
             <ul class="nav">
-                <li><a href="sign-in.html">Sign In</a></li>
-                <li><a href="sign-up.html">Sign Up</a></li>
```

Comment:
No need to delete this

Reply:
like how you changed sign in into Log in

Reply:
that needs to be deleted. Because there's no more sign-in.html there.s a login.html though